### DISCUSSION TITLE
镜像创建 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1073] [Announcements]
# 镜像创建 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/workload/deployment/

**Comment by liyuerich** [comment]
可以选择厂商、地域、可用区三种：
每一种能有个图示介绍吗？或者具体在哪里进行配置后才能看见
在文档站上没有找到

**Reply by Michelle951** [reply]
文档中暂时没有(后续会补充），可参考[视频教程](https://docs.daocloud.io/videos/kairship/#_4) 从 00:30 开始，有关标签操作符的说明可参考<https://docs.daocloud.io/middleware/minio/user-guide/create/>步骤5下的高级配置说明
<img width="836" alt="image" src="https://user-images.githubusercontent.com/35519562/226869658-2d1758bc-83c0-4f83-a807-00bccba4d939.png">
